lib/messager.rb

```diff
@@ -0,0 +1,18 @@
+require "dotenv/load"
```

Comment:
No tests for this class! 😱 

Testing Twilio properly is probably the hardest part of this whole challenge - it really tests your skills with mocking and stubbing.  I think Sophie went over a possible implementation on Monday?  It might be worth going back to this now and seeing if you can get it working if you're interested?